sql/src/main/java/io/crate/executor/transport/TransportShardAction.java

```diff
@@ -107,7 +106,8 @@ public Tuple call() throws Exception {
             //noinspection unchecked
             response = callable.call();
         } catch (Throwable e) {
-            throw Throwables.propagate(e);
+            assert e instanceof RuntimeException : "Expecting instance of RuntimeException here";
```

Comment:
I don't get it =)
Throwables.propagate(e) doesn't wrap the exception if it already is a RuntimeException